### PR TITLE
Closes #127

### LINKED ISSUE #127: [DOCS] Make installation step 5 clearer
### Description

At the moment instruction 5 says: Add a GitHub Token in .env (Copy .env.example and paste credentials)

It would be clearer is it instead says: Create a .env file in the root of the project. Then add your GitHub token (see example in .env.example file)

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by frazie**
## Fixes Issue
Closes #127

## Check List (Check all the applicable boxes) 

- [x ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [x ] I have updated the documentation accordingly.
- [x ] All new and existing tests passed.
- [x ] This PR does not contain plagiarized content.
- [ x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/good-first-issue-finder/pull/129"></a>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ Before installation, please make sure you have already installed the following t
     ```
 3. Navigate to the project directory `cd good-first-issue-finder`
 4. Install dependencies with `npm install`
-5. Add a GitHub Token in .env (Copy .env.example and paste credentials)
+5. Create a .env file in the root of the project. Then add your GitHub token (see example in .env.example file)
```
Comment (schmelto) [inline README.md:35]:
```suggestion
5. Create a `.env` file in the root of the project. Then add your GitHub token (see example in `.env.example` file)
```

**Review by emmalearnscode — APPROVED** [review]
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

File: README.md (hunk 1 above)
Comment (Cahllagerfeld) [reply README.md:35]: I think we should utilize these inline-blocks, we're good to go once the suggestions are commited

**Review by Cahllagerfeld — APPROVED** [review]
Looks good to me 👍

**Pull request merged**